usr.sbin/nfsd/nfsd.c

```diff
@@ -176,23 +176,32 @@ main(int argc, char **argv)
 	char **bindhost = NULL;
 	pid_t pid;
 	struct nfsd_nfsd_args nfsdargs;
+	const char *vhostname = NULL;
 
 	nfsdargs.mirrorcnt = 1;
 	nfsdargs.addr = NULL;
 	nfsdargs.addrlen = 0;
 	nfsdcnt = DEFNFSDCNT;
 	unregister = reregister = tcpflag = maxsock = 0;
 	bindanyflag = udpflag = connect_type_cnt = bindhostc = 0;
-	getopt_shortopts = "ah:n:rdtuep:m:";
+	getopt_shortopts = "ah:n:rdtuep:m:V:";
 	getopt_usage =
 	    "usage:\n"
 	    "  nfsd [-ardtue] [-h bindip]\n"
 	    "       [-n numservers] [--minthreads #] [--maxthreads #]\n"
 	    "       [-p/--pnfs dsserver0:/dsserver0-mounted-on-dir,...,"
 	    "dsserverN:/dsserverN-mounted-on-dir] [-m mirrorlevel]\n";
+	    "       [-V virtual_hostname]\n"
```

Comment:
You need to move the semicolon now too.

Reply:
Fixed, and did a buildworld just to be sure :).

> new%  ./nfsd -:
> nfsd: invalid option -- :
> usage:
>  nfsd [-ardtue] [-h bindip]
>       [-n numservers] [--minthreads #] [--maxthreads #]
>       [-p/--pnfs dsserver0:/dsserver0-mounted-on-dir,...,dsserverN:/dsserverN-mounted-on-dir] [-m mirrorlevel]
>       [-V virtual_hostname]
> old% /usr/sbin/nfsd -:
> nfsd: invalid option -- :
> usage:
>   nfsd [-ardtue] [-h bindip]
>        [-n numservers] [--minthreads #] [--maxthreads #]
>        [-p/--pnfs dsserver0:/dsserver0-mounted-on-dir,...,dsserverN:/dsserverN-mounted-on-dir] [-m mirrorlevel]